### DISCUSSION TITLE
Limit content based on access

### DISCUSSION THREAD

**Discussion by ellanrk** [microsoft-search/pnp-modern-search#3652] [Q&A] [answered]
Is there a way to show only the files/folders that logged-in user has access to edit?

Current search query template: 
_{searchTerms} -SPSiteURL:https://domain-my*_

**Comment by kasperbolarsen** [comment, marked as answer]
If "edit" isn't a typo, then the answer is a no, search is on access to the object, so we can not tell if the user has read or edit permissions.

**Reply by ellanrk** [reply]
Gotcha! It wasn't typo. I thought there is no way but wanted to confirm.
